ui/app/components/namespace-picker.ts

```diff
@@ -75,45 +70,33 @@ export default class NamespacePicker extends Component {
     return options.find((option) => this.matchesPath(option, currentPath));
   }
 
-  private getOptions(
-    accessibleNamespaces: string[],
-    currentNamespace: string,
-    path: string
-  ): NamespaceOption[] {
-    /* Each namespace option has 3 properties: { id, path, and label }
-     *   - id: node / namespace name (displayed when the namespace picker is closed)
+  private getOptions(accessibleNamespaces: string[]): NamespaceOption[] {
+    /* Each namespace option has 2 properties: { path and label }
```

Comment:
The only reason we can't just return a list of `paths` is because the root path is `""` and we want the label to be `root`. 

Even though it's more verbose, it seemed clearer to manage the options as an object rather than manually handling the explicit root case in the dropdown and `onChange` function. This also makes it more extensible if we ever decide to leverage namespace metadata here somehow.